scripts/_helpers_dist.py

```diff
@@ -2,16 +2,26 @@
 # SPDX-FileCopyrightText: : 2017-2020 The PyPSA-Eur Authors
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
+import logging
 import os
+import sys
 from pathlib import Path
 
+sys.path.append("../pypsa-earth/scripts")
+sys.path.append("pypsa-distribution/pypsa-earth/scripts")
```

Comment:
Can you revise this as follows? The CI is not working and I believe the problem is this one
```suggestion
sys.path.append("./pypsa-earth/scripts")
```